tests/util/kube_utils.go

```diff
@@ -101,44 +101,44 @@ func CreateNamespace(n string) error {
 }
 
 // DeleteNamespace delete a kubernetes namespace
-func DeleteNamespace(n string) error {
-	_, err := Shell("kubectl delete namespace %s", n)
+func DeleteNamespace(n string, kubeconfig string) error {
+	_, err := Shell("kubectl delete namespace %s --kubeconfig=%s", n, kubeconfig)
 	return err
 }
 
 // NamespaceDeleted check if a kubernete namespace is deleted
-func NamespaceDeleted(n string) (bool, error) {
-	output, err := ShellSilent("kubectl get namespace %s -o name", n)
+func NamespaceDeleted(n string, kubeconfig string) (bool, error) {
+	output, err := ShellSilent("kubectl get namespace %s -o name --kubeconfig=%s", n, kubeconfig)
 	if strings.Contains(output, "NotFound") {
 		return true, nil
 	}
 	return false, err
 }
 
 // KubeApplyContents kubectl apply from contents
-func KubeApplyContents(namespace, yamlContents string) error {
+func KubeApplyContents(namespace, yamlContents string, kubeconfig string) error {
```

Comment:
I think if your not going to pass the appManager struct on all of these operations, it makes more sense to order every operation as
eg:
(namespace, kubeconfig, ...)

This current patch orders everything as
eg:
(namespace, ... , kubeconfig).

This is just a stylistic issue - I generally think related variables should be grouped and ordered in the prototype (unless there is some strong rationale not to do that (i.e. would break an ABI).  May not be worth the trouble to change it around.  Leave to others to decide.

Cheers
-steve